recipes/mini/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+
+#include "mini/ini.h"
+
+int main(void) {
+    mINI::INIFile file("test_package.ini");
```

Comment:
@RubenRBS  I've updated the test_package - its a header only. library, so no need to check for linkage, and the constructor does not write a file to disk (it does nothing).